Core/Core/Models/GraphTraversal/DefaultTraversal.cs

```diff
@@ -86,35 +57,28 @@ public static GraphTraversal CreateBIMTraverseFunc(ISpeckleConverter converter)
       .When(converter.CanConvertToNative)
       .ContinueTraversing(ElementsAliases);
 
-    //WORKAROUND: ideally, traversal rules would not have Objects specific rules.
-    var ignoreResultsRule = TraversalRule
-      .NewTraversalRule()
-      .When(o => o.speckle_type.Contains("Objects.Structural.Results"))
-      .ContinueTraversing(None);
-
-    var defaultRule = TraversalRule.NewTraversalRule().When(_ => true).ContinueTraversing(Members());
-
-    return new GraphTraversal(bimElementRule, ignoreResultsRule, defaultRule);
+    return new GraphTraversal(bimElementRule, IgnoreResultsRule, DefaultRule);
   }
 
   //These functions are just meant to make the syntax of defining rules less verbose, they are likely to change frequently/be restructured
-
   #region Helper Functions
 
-  internal static readonly string[] elementsAliases = { "elements", "@elements" };
-
-  internal static IEnumerable<string> ElementsAliases(Base _)
-  {
-    return elementsAliases;
-  }
+  //WORKAROUND: ideally, traversal rules would not have Objects specific rules.
```

Comment:
Agreed! We can live with this for now, but if some Objects need to be ignored we may want to create a specific class attribute for it.

I'll open a separate issue with this.